content/docs/rest-helix/users-get.md

```diff
@@ -57,7 +57,10 @@ resultList.getUsers().forEach(user -> {
 {{</code>}}
 {{<code Kotlin>}}
 ```kotlin
-
+val resultList = twitchClient!!.helix.getUsers(null, listOf("149223493"), null).execute()
```

Comment:
```suggestion
val resultList = twitchClient.helix.getUsers(null, listOf("149223493"), null).execute()
```
we can also remove the two `!!` as we don't have this in the other examples.

Reply:
there is no need to be a null safety / null checks